l5kit/l5kit/random/random_generator.py

```diff
@@ -55,3 +55,20 @@ def _sample(self) -> np.ndarray:
 
     def __call__(self) -> np.ndarray:
         return self._sample()
+
+
+class UniformRandomGenerator:
```

Comment:
This would be better sub-classing a `RandomGenerator` class, as we're doing type hinting using Python's ABC as well.